preload/avoid_delaying_onload_link_preload.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    var t = async_test('Makes sure link preload preloaded resources are not delaying onload');
+</script>
+<link rel=preload href="resources/dummy.js?pipe=trickle(d1)" as=script>
```

Comment:
Do you think the time it'd take the browser to run the test and get to window.load() would exceed 1 second? I'd hate to make this test slower if it's not a must